### ISSUE TITLE
Current development state ?

### ISSUE THREAD

**Issue by mStirner** [NodeOS/NodeOS#414]
What is the current development state of this repo ?

The last effective commit looks like is 2 years ago.
Is this project til maintained ?

**Comment by piranna** [comment]
Yes and no... I have still plans for it, but don't have time to work on it due to work and other projects. I tried to find a maintainer or make it community driven, but failed to do it so :-(

**Comment by mStirner** [comment]
If there would be a better documentation im 100% sure more people would contribute to this project.

It is very hard for beginners to understand how NodeOS works (me included)

**Comment by sekaiser** [comment]
As a starting point, does it make sense to address concrete questions in the form of issues? This way, the community may step in contributing content which may later be included into the docs.

**Comment by OrdinaryJellyfish** [comment]
I think I could help with documentation (and possibly some development of NodeOS). I've seen how NodeOS works and think it would be an amazing OS for servers that are simply for Node.js applications and maybe even a few other things. I definitely agree with @mStirner, a lot more people would most likely be interested in the project if it had better and easier to understand documentation about using and extending NodeOS.

**Comment by soydepr** [comment]
Can I install on Mac using virtual box or parallels

**Comment by AndreiLeman** [comment]
> Can I install on Mac using virtual box or parallels

I suppose you can, same as with any other OSes

**Comment by RossComputerGuy** [comment]
@piranna If your wanting someone else to take over/help with running the project, I'll be getting enough free time during the summer and I'll be willing to run it.

**Comment by stevenvachon** [comment]
I'd be interested in helping with a GUI if Chrome were bundled in somehow.

**Comment by damusix** [comment]
This looks like fun. I'll join the bandwagon.



> I'd be interested in helping with a GUI if Chrome were bundled in somehow.

@stevenvachon How about electron? more better, no?

**Comment by stevenvachon** [comment]
@damusix I'm unsure if it's better or not to interface with Node.js via a network protocol instead of without limitation via Electron.

**Edit:** another possibility is the [extensions API](https://developer.chrome.com/extensions) or WebAssembly.

**Comment by mStirner** [comment]
We date the year 2025.
Any statement ?

**Comment by ninjaaron** [comment]
@mStirner The last commit was 7 years ago. Sort of speaks for itself.
